### DISCUSSION TITLE
Lucene/Search/2024/0829/queryCache/

### DISCUSSION THREAD

**Discussion by giscus** [LuXugang/Lucene-7.x-9.x#45] [Announcements]
# Lucene/Search/2024/0829/queryCache/

Hexo Theme Keep

https://www.amazingkoala.com.cn/Lucene/Search/2024/0829/queryCache/

**Comment by RockNHawk** [comment]
请问大佬，如果给index searcher的不是单一查询，而是boolean query内，套了多个子查询，期中有phrase query，自定义的query等等，Lucene会把缓存颗粒度细到子query吗？
